src/components/Calculator.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+import React from 'react';
+import '../App.css';
+import Buttons from './Buttons';
+import Output from './Output';
+
+// eslint-disable react/prefer-stateless-function
```

Comment:
- [ ] [ OPTIONAL]  KIndly fix linter issue manually after removing these lines. This is going mandatory requirement in further process.